### ISSUE TITLE
something wrong from def upload_pins(self, pin: int) -> None:

### ISSUE THREAD

**Issue by noellesugar99** [maximedrn/pinterest-automatic-upload#8]
Login to Pinterest. Logged.

Uploading pins n°1/1. Failed. Message: 
Stacktrace:
0   chromedriver                        0x000000010547cfa8 chromedriver + 4886440
1   chromedriver                        0x00000001053fa643 chromedriver + 4351555
2   chromedriver                        0x0000000105048b27 chromedriver + 477991
3   chromedriver                        0x000000010508d76c chromedriver + 759660
4   chromedriver                        0x000000010508d9c1 chromedriver + 760257
5   chromedriver                        0x00000001050d1204 chromedriver + 1036804
6   chromedriver                        0x00000001050b35fd chromedriver + 914941
7   chromedriver                        0x00000001050ce5fe chromedriver + 1025534
8   chromedriver                        0x00000001050b33a3 chromedriver + 914339
9   chromedriver                        0x000000010507d57f chromedriver + 693631
10  chromedriver                        0x000000010507eb1e chromedriver + 699166
11  chromedriver                        0x0000000105449b9e chromedriver + 4676510
12  chromedriver                        0x000000010544e91e chromedriver + 4696350
13  chromedriver                        0x000000010545619f chromedriver + 4727199
14  chromedriver                        0x000000010544f81a chromedriver + 4700186
15  chromedriver                        0x0000000105422a62 chromedriver + 4516450
16  chromedriver                        0x000000010546e8c8 chromedriver + 4827336
17  chromedriver                        0x000000010546ea45 chromedriver + 4827717
18  chromedriver                        0x00000001054847ef chromedriver + 4917231
19  libsystem_pthread.dylib             0x00007ff80c78d4e1 _pthread_start + 125
20  libsystem_pthread.dylib             0x00007ff80c788f6b thread_start + 15

hi i hope you can help me

**Comment by maximedrn** [comment]
When does this happen?
It comes from a wrong XPATH in the script that causes the crash

**Comment by noellesugar99** [comment]
Thank you for your reply. 
I did it now again in case and this is what i got. I also attached a schreenshot video

Login to Pinterest. Logged.

Uploading pins n°1/1. Failed. Message: 
Stacktrace:
0   chromedriver                        0x000000010467bfa8 chromedriver + 4886440
1   chromedriver                        0x00000001045f9643 chromedriver + 4351555
2   chromedriver                        0x0000000104247b27 chromedriver + 477991
3   chromedriver                        0x000000010428c76c chromedriver + 759660
4   chromedriver                        0x000000010428c9c1 chromedriver + 760257
5   chromedriver                        0x00000001042d0204 chromedriver + 1036804
6   chromedriver                        0x00000001042b25fd chromedriver + 914941
7   chromedriver                        0x00000001042cd5fe chromedriver + 1025534
8   chromedriver                        0x00000001042b23a3 chromedriver + 914339
9   chromedriver                        0x000000010427c57f chromedriver + 693631
10  chromedriver                        0x000000010427db1e chromedriver + 699166
11  chromedriver                        0x0000000104648b9e chromedriver + 4676510
12  chromedriver                        0x000000010464d91e chromedriver + 4696350
13  chromedriver                        0x000000010465519f chromedriver + 4727199
14  chromedriver                        0x000000010464e81a chromedriver + 4700186
15  chromedriver                        0x0000000104621a62 chromedriver + 4516450
16  chromedriver                        0x000000010466d8c8 chromedriver + 4827336
17  chromedriver                        0x000000010466da45 chromedriver + 4827717
18  chromedriver                        0x00000001046837ef chromedriver + 4917231
19  libsystem_pthread.dylib             0x00007ff80c78d4e1 _pthread_start + 125
20  libsystem_pthread.dylib             0x00007ff80c788f6b thread_start + 15



https://user-images.githubusercontent.com/115692127/216971738-7d083844-9c3e-4ed8-8a2d-82dafa736212.mov
